src/client.js

```diff
@@ -196,7 +196,7 @@ export default class Client {
     const [query, path] = node.nextPageQueryAndPath();
     let variableValues;
 
-    if (variableValues || options) {
+    if (node.variableValues || options) {
```

Comment:
this was a typo, and there was no test that didn't submit variable values, so the case of no new variables + old variables wasn't covered.